dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.40/V2_40_10__Create_table_program_stage_working_list_definition.sql

```diff
@@ -0,0 +1,24 @@
+CREATE TABLE if not exists programstageworkinglistdefinition (
+    programstageworkinglistdefinitionid integer NOT NULL,
+    uid character varying(11) NOT NULL,
+    code character varying(50),
+    created timestamp without time zone NOT NULL,
+    lastupdated timestamp without time zone NOT NULL,
+    lastupdatedby integer,
```

Comment:
You are missing "CreatedBy", its defined in the hbm file, but not here :)

Reply:
I just added it, but I see we are actually not using it in any of the other filters, should I also not add it in this one?